launcher.py

```diff
@@ -0,0 +1,30 @@
+# launcher.py
+
+from bot.bot import LeagueBot
+
+import asyncio
+import asyncpg
+from dotenv import load_dotenv
+from operator import itemgetter
+from os import environ
+
+
+def run_bot():
+    """ Parse the config file and run the bot. """
+    # Load the environment variables in the local .env file
+    load_dotenv()
+
+    # Get database object for bot
+    connect_url = 'postgresql://{POSTGRESQL_USER}:{POSTGRESQL_PASSWORD}@{POSTGRESQL_HOST}/{POSTGRESQL_DB}'
+    loop = asyncio.get_event_loop()
+    db_pool = loop.run_until_complete(asyncpg.create_pool(connect_url.format(**environ)))
+
+    # Instantiate bot and run
+    dbl_token = environ.get('DBL_API_KEY', None)
```

Comment:
DBL?

Reply:
It's the API for a website where you can advertise Discord bots. Sorta leftover from the original repo. I guess we don't actually need it here.

Reply:
https://top.gg/bot/539669626863353868

Reply:
I'd say it might not be worth keeping in this fork